heartbeat/docs/monitors/monitor-common-options.asciidoc

```diff
@@ -54,22 +52,22 @@ default.
 
 A cron-like expression that specifies the task schedule. For example:
 
-* `*/5 * * * * * *` runs the task every 5 seconds (for example, at 10:00:00,
+* `*/5 * * * * * *` runs the task every five seconds (for example, at 10:00:00,
 10:00:05, and so on).
-* `@every 5s` runs the task every 5 seconds from the time when {beatname_uc} was
+* `@every 5s` runs the task every five seconds from the time when {beatname_uc} was
 started.
 
 The `schedule` option uses a cron-like syntax based on https://github.com/gorhill/cronexpr#implementation[this `cronexpr` implementation],
 but adds the `@every` keyword.
 
-For stats on the execution of scheduled tasks you can enable the HTTP stats server with `http.enabled: true` in heartbeat.yml, then run `curl http://localhost:5066/stats | jq .heartbeat.scheduler` to view the scheduler's stats. Stats are provided for both jobs and tasks. Each time a monitor is scheduled is considered to be a single job, while portions of the work a job does, like DNS lookups and executing network requests are defined as tasks. The stats provided are:
+For stats on the execution of scheduled tasks, enable the HTTP stats server with `http.enabled: true` in heartbeat.yml. Then, run `curl http://localhost:5066/stats | jq .heartbeat.scheduler` to view the scheduler's stats. Stats are provided for both jobs and tasks. Each time a monitor is scheduled, it is considered to be a single job. DNS lookups and executing network requests are defined as tasks. The stats provided are:
 
 * **jobs.active:** The number of actively running jobs/monitors.
-* **jobs.missed_deadline:** The number of jobs that executed after their scheduled time. This can be caused either by overlong long timeouts from the previous job or high load preventing heartbeat from keeping up with work.
+* **jobs.missed_deadline:** The number of jobs that are executed after their scheduled time. This can be caused either by overlong long timeouts from the previous job or high load preventing heartbeat from keeping up with work.
```

Comment:
I think you want "that have executed" here. Otherwise, you are changing the meaning slightly.